folium/features.py

```diff
@@ -582,6 +582,34 @@ class GeoJson(Layer):
                     }
                 },
                 {%- endif %}
+
+                {%- if this.highlight and this.popup_keep_highlighted %}
```

Comment:
Is it better if we merge the two declarations of mouseout and mouseover? Then we would have less duplicate code. For example:

```
{%- if this.highlight %}
mouseout: function(e) {
    if(typeof e.target.setStyle === "function"){
        {%- if this.popup_keep_highlighted %}
        if (!{{ this.get_name() }}.isPopupOpen())
        {%- endif %}
            {{ this.get_name() }}.resetStyle(e.target);
    }
},
mouseover: function(e) {
    if(typeof e.target.setStyle === "function"){
        const highlightStyle = {{ this.get_name() }}_highlighter(e.target.feature)
        e.target.setStyle(highlightStyle);
    }
},
{%- endif %}
```